src/requirements/data/templates/cs-eng.ts

```diff
@@ -0,0 +1,87 @@
+import { PlaceholdersForRequirement } from '@/requirements/types';
+
+const csEngTemplate: readonly PlaceholdersForRequirement[] = [
```

Comment:
Maybe defining something like `type Template = readonly PlaceholdersForRequirement[]` and `const csEngTemplate: Template` will make this more clear?